recipes/multimedia/gstreamer.yaml

```diff
@@ -0,0 +1,35 @@
+inherit: [meson]
+
+metaEnvironment:
+   PKG_VERSION: "1.18.2"
+
+depends:
+   - libs::glib-dev
+   - use: []
+     depends:
+        - libs::glib-tgt
+
+checkoutSCM:
+   -
+    scm: url
+    url: https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-${PKG_VERSION}.tar.xz
+    digestSHA1: 2a417e7c5eb184b705e0964bb63f83f1498bdf84
+
+buildVars: [PKG_VERSION]
+buildTools: [flex, bison]
+buildScript: |
+   export BISON_PKGDATADIR="${BOB_TOOL_PATHS[bison]}/../share/bison"
+   mesonAddTool bison "${BOB_TOOL_PATHS[bison]}/bison-real"
+   mesonAddTool flex "${BOB_TOOL_PATHS[flex]}/flex"
```

Comment:
That looks odd. According to the documentation tools are searched in `$PATH` automatically. Also, why is it necessary to re-do the bison wrapper script here?

Reply:
Without this `gstreamer` build fails cause `bison` and `flex` can't be found. Directly executing the bison wrapper also failed with `exec format error`.

Reply:
would it be possible to extend bob in order to remove the wrapper by using 

```
provideVars:
 BISON_PKGDATADIR: "$SOME_COOL_BOB_VAR/../share/bison"
```

in the `bison.yaml`?

Reply:
No, that won't work because all variables are computed independent of the build location.

I'm currently testing your changes. The `exec format error` is caused by a missing `#!/bin/sh` header of the wrapper script. Unfortunately this leads to a rebuild of *everything*. :sob: